packages/loot-core/src/server/budget/app.ts

```diff
@@ -28,6 +28,10 @@ app.method(
   'budget/overwrite-goal-template',
   mutator(undoable(goalActions.overwriteTemplate)),
 );
+app.method(
```

Comment:
This was a convenient place to put this.  I'm not sure if it's the correct place.